src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -114,27 +123,60 @@ class ActiveFocusInvestigation extends React.Component<
       },
       pages: [],
     };
+
+    const basePage = {
+      label: FOCUS_INVESTIGATIONS,
+      url: FI_URL,
+    };
     const homePage = {
-      label: `${HOME}`,
-      url: `${HOME_URL}`,
+      label: HOME,
+      url: HOME_URL,
     };
-    breadcrumbProps.pages = [homePage];
 
-    const { caseTriggeredPlans, routinePlans } = this.props;
+    const { caseTriggeredPlans, routinePlans, plan } = this.props;
+    // We need to initialize jurisdictionName to a falsy value
+    let jurisdictionName = null;
+
+    // get the current page index
+    if (plan) {
+      const currentPageIndex: number = plan.jurisdiction_path.indexOf(
+        this.props.match.params.jurisdiction_parent_id
+      );
+      breadcrumbProps.currentPage = {
+        label: plan.jurisdiction_name_path[currentPageIndex],
+        url: '',
+      };
+      jurisdictionName = plan.jurisdiction_name_path[currentPageIndex];
+      const labels = plan.jurisdiction_name_path.slice(0, currentPageIndex);
+      breadcrumbProps.pages = labels.map((label, i) => {
+        return {
+          label,
+          url: FI_URL + '/filter/' + plan.jurisdiction_path[i] + '/' + plan.id,
```

Comment:
Let's make this in a getter or at least don't concatenate

Reply:
This was already fixed

Reply:
If we were to use a getter here @cKellyDesign, where(in what file) would it live?

Reply:
@FrankApiyo I would think `src/helpers/utils.tsx` or a new file in the helpers directory that is for such types of getters (maybe just in utils for now though)